integration-tests/pom.xml

```diff
@@ -50,5 +50,23 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
```

Comment:
This is not actually used in one IT module: ws-security-server

But I don't see any real harm defining that very commong dep up here.

Reply:
I fact, we could pull up even more deps like testcontainers...